### PR TITLE
TextField & SearchBox: Specified placeholder colors

### LINKED ISSUE #3756: [Search Box] The search box text appears dark black on IE 11 as compared to other browsers like chrome
### Bug Report

- __Package version(s)__: 5.4.1
- __Browser and OS versions__: IE 11 Version (11.192.16299.0)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: Internal Sites

#### Describe the issue:

The search box text appears dark black on IE 11 as compared to other browsers like chrome

#### Actual behavior:

This snapshot is taken from IE 11
image

#### Expected behavior:

This is snapshot is taken from chrome 
image

### PR DESCRIPTION
**Pull request by lynamemi**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #3756
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Allows us to be consistent across browsers

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { ISearchBoxProps, ISearchBoxStyleProps, ISearchBoxStyles } from './Searc
 
 export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
   const { theme, underlined, disabled, hasFocus, className, hasInput } = props;
-  const { palette, fonts } = theme;
+  const { palette, fonts, semanticColors } = theme;
```
Comment (phkuo) [inline packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx]: shouldn't you get a compiler warning for adding semanticColors here but not use it?

File: packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx [hunk 2]
```diff
@@ -142,6 +142,16 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
...
             display: 'none'
+          },
+          '::placeholder': {
+            color: palette.neutralSecondary,
+            opacity: 1
+          },
+          ':-ms-input-placeholder': {
+            color: palette.neutralSecondary
+          },
+          '::-ms-input-placeholder': {
```
Comment (micahgodbolt) [inline packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx]: are both syntaxes necessary?

Comment (lynamemi) [reply packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx]: Apparently browser versioning and placeholder selectors get hairy, but it looks like using just the two will work on at at least all my browser versions.

File: packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx (hunk 1 above)
Comment (lynamemi) [reply packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx]: true! I just added the slot for inputPlaceholderText.

**Pull request merged**